src/metabase/models/dashboard.clj

```diff
@@ -191,7 +191,7 @@
             current-card    (id->current-card dashcard-id)]
         (cond
           ;; If card is in current-cards but not serialized-cards then we need to delete it
-          (not serialized-card) (dashboard-card/delete-dashboard-card! current-card user-id)
+          (not serialized-card) (dashboard-card/delete-dashboard-card! (:id current-card))
```

Comment:
previously the `dashboard-card/delete-dashboard-card!` published an event that caused new revisions to be recorded.